lib/queuePopulator/LogReader.js

```diff
@@ -252,6 +252,7 @@ class LogReader {
             readOptions.limit = params.maxRead;
         }
         this.log.debug('reading records', { readOptions });
+        // need to implement this method for mongo!!
```

Comment:
what happens if it's not implemented?

Reply:
Mmmmm I think this was a residual comment before the completion of this: https://github.com/scality/Arsenal/blob/master/lib/storage/metadata/mongoclient/LogConsumer.js
:sweat_smile: